core/Cargo.toml

```diff
@@ -156,6 +156,7 @@ services-gcs = [
 ]
 services-gdrive = ["internal-path-cache"]
 services-ghac = []
+services-github-contents = []
```

Comment:
Maybe use `github` is enough? Will we support other services for github?

Reply:
fixed